aix/README.md

```diff
@@ -5,39 +5,7 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 
 ## Building RPM packages
```

Comment:
RPM Packages? Or AIX packages?

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.